samples/client/petstore/java/retrofit/src/test/java/io/swagger/client/api/PetApiTest.java

```diff
@@ -76,33 +76,6 @@ public void testFindPetsByStatus() throws Exception {
     }
 
     @Test
-    public void testFindPetsByTags() throws Exception {
```

Comment:
@giuliopulina May I know why the test cases for `findPetsByTags` are removed? Is it throwing errors when run locally?

Reply:
yes, I confirm that mvn integration-test was throwing error.

In fact, if I try to call the service: http://petstore.swagger.io/v2/pet/findByTags?tags=friendly

I receive this error:

      <apiResponse>
          <message>something bad happened</message>
          <type>unknown</type>
      </apiResponse>

And also, if I look in swagger UI, I see that this endpoint is deprecated:

![image](https://user-images.githubusercontent.com/4653848/31993660-364bdd84-b97e-11e7-82f5-9c8433a320b6.png)

Reply:
The CIs (Travis, CircleCI) are running a local copy of the Petstore server as the public petstore server contains invalid pet data from time to time (since anyone can update the record).

Please add those tests back and the CI tests should pass.

Reply:
done 👍